assets/src/scripts/components/dailyValueHydrograph/index.js

```diff
@@ -45,13 +42,16 @@ export const attachToNode = function (store,
                 });
             } else {
                 store.dispatch(Actions.setCurrentObservationsTimeSeriesId(timeSeriesId));
+
+                nodeElem.select('.graph-container')
+                    .call(drawTimeSeriesGraph, store)
+                    .call(drawTooltipCursorSlider, store);
+
+                // Add DV legend
+                let graphContainer = nodeElem.select('.graph-container');
```

Comment:
If you move this to above line 46 and then on line 46 you can just have "graphContainer". This will make the code more dry.